counsel.el

```diff
@@ -4138,12 +4142,7 @@ current outline heading in emacs-lisp-mode buffers. See
 (defun counsel-outline ()
   "Jump to outline with completion.
 
-This command relies on `outline-regexp', `outline-level', and
-`counsel-outline-title' to match an outline heading and get its
-level and title, respectively.  For major modes where this is not
-adequate or optimal, these variables can be rebound locally in
-the major mode hook.  Replacements are provided for some such
-modes. To set them up, add this to your init file:"
+This command's behavior can be customized based on the current buffer's major mode.  See variable `counsel-outline-settings'."
```

Comment:
Needs more filling! :)

Reply:
Not sure what you mean. I agree the sentence is not very informative, but on the other hand I didn't want to duplicate the information of `counsel-outline-settings`'s docstring.

So I just removed the sentence. This actually seems in line with many counsel commands that have short docstrings not mentioning the corresponding custom variables.

But of course if you have suggestions, @abo-abo will make the decision.